### ISSUE TITLE
Charts data can't be read by VoiceOver

### ISSUE THREAD

**Issue by irfano** [wordpress-mobile/WordPress-iOS#20007]
### Expected behavior
The points on bar charts and line charts should be readable by VoiceOver.

### Actual behavior
The points on bar charts and line charts can't be selected, and VoiceOver can't read them.

<img src="https://user-images.githubusercontent.com/2471769/214884683-e3a5b2f0-0f9f-4a7c-92fd-53c0897c310a.PNG" height=300>
<img src="https://user-images.githubusercontent.com/2471769/214884678-9200ce79-42c4-42aa-9dc0-62e49ac96c7b.PNG" height=300>

### Steps to reproduce the behavior
1. Activate VoiceOver from device Settings → Accessibility.
2. Open the JP app and log in.
3. Open "My Site → Stats".
4. Try to select points on the line chart of "Views & Visitors" card. Notice they can't be selected, and VoiceOver can't read them.
5. Navigate to "DAYS" tab.
4. Try to select points on the bar. Notice they can't be selected, and VoiceOver can't read them.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
